doc/deployment_guide.md

```diff
@@ -0,0 +1,6080 @@
+
+The high-level steps required to install the Dell EMC Ready Architecture for Red Hat OpenStack Platform v13 using the automated installation procedures include:
+
+1.  Ensuring that your environment meets the [Prerequisites](#Prerequisites)
+2.  Ensuring that the [Dependencies](#Dependencies) on are met.
+3.  *Determining Pool IDs*.
+4.  [Downloading and Extracting Automation Files](#page12).
+5.  [Preparing the Solution Admin Host Deployment](#page14).
+6.  [Deploying the SAH Node](#page16).
+7.  [Deploying the Undercloud and the OpenStack Cluster](#page18).
+
+## Prerequisites
+The following prerequisites must be satisfied before proceeding with a Dell EMC Ready Architecture for Red Hat OpenStack platform v13.0 deployment:
+
+> Note: All nodes in the same roles must be of the same server models, with identical HDD, RAM, and NIC configurations. So, all Controller nodes must be identical to each other; all Compute nodes must be identical to each other; and so on. See the Dell EMC Ready Architecture for Red Hat OpenStack Platform - Version 13 for configuration options for each node role.
+
+* Hardware racked and wired per the [Dell EMC Ready Architecture for Red Hat OpenStack Platform Architecture Guide Version 13](https://www.dellemc.com/resources/en-us/asset/technical-guides-support-information/solutions/dell_emc_ready_architecture_for_red_hat_openstack_platform_architecture_guide.pdf).
+* Hardware configured as per the [Dell EMC Ready Architecture for Red Hat OpenStack Platform Architecture Guide Version 13](https://www.dellemc.com/resources/en-us/asset/technical-guides-support-information/solutions/dell_emc_ready_architecture_for_red_hat_openstack_platform_architecture_guide.pdf).
+* Hardware is powered off after the hardware is configured per the [Dell EMC Ready Architecture for Red Hat OpenStack Platform Architecture Guide Version 13](https://www.dellemc.com/resources/en-us/asset/technical-guides-support-information/solutions/dell_emc_ready_architecture_for_red_hat_openstack_platform_architecture_guide.pdf).
+* Internet access, including but not limited to, Red Hat’s subscription manager service and repositories
+* Valid Red Hat subscriptions
+* Workstation used to extract the JetPack-automation-13.0.tgz file and begin building the collateral for the SAH node.Workstation must be a RHEL7.6 host.
+
+## Dependencies
+For customers performing a self-installation, these files are available upon request from Dell EMC. Please contact your account representative, or email <a href="malito: openstack@dell.com" target="_blank">openstack@dell.com</a> for instructions.
+
+> NOTE: The files are also open sourced and can be obtained from <a href="https://github.com/dsp-jetpack/JetPack." target="_blank">https://github.com/dsp-jetpack/JetPack.</a> The Dell EMC Ready Architecture for Red Hat OpenStack Platform v13 deployment dependencies include
+

```

Comment:
update to https://github.com/dsp-jetpack/JetPack/tree/JS-13.0

Reply:
updated